README.markdown

```diff
@@ -6,6 +6,15 @@
 
 # Sippy Cup
 
+## NOTES for this fork
+
+* New instruction is added to insert wav file into pcap
+  -  `- play_audio "<wav file path>"`
+  - need spandsp library for encoidng a-law and u-law
+* Debug the DTMF packet generation (end of event) 
+  - reduce the duration to 200 milliseconds
+  - change obsolete rfc2833 to rfc4733
```

Comment:
To merge this into the main branch, would you move these notes into appropriate areas of the README?